src/components/InputNumber/InputNumber.tsx

```diff
@@ -117,16 +145,18 @@ function InputNumber({
             min={min}
             max={max}
           />
-          <div className={InputNumberStyles['sbui-inputnumber-nav']}>
-            <IconChevronUp
-              className={iconUpClasses.join(' ')}
-              onClick={onClickChevronUp}
-            />
-            <IconChevronDown
-              className={iconDownClasses.join(' ')}
-              onClick={onClickChevronDown}
-            />
-          </div>
+          {isFocused ? (
```

Comment:
It might make more sense to use `.sbui-inputnumber:focus` css rather than using a react state.

for example

```css

/* InputNumber.module.css */

.sbui-inputnumber-nav {
  display:none;
}

.sbui-inputnumber:focus .sbui-inputnumber-nav {
  display:block;
}

/* or with tailwind */

.sbui-inputnumber-nav {
  @apply hidden;
}

.sbui-inputnumber:focus .sbui-inputnumber-nav {
 @apply block;
}
```

can probably remove the useEffect method above then as well

Reply:
yup done and pushed the changes